index.rst

```diff
@@ -1,61 +1,1351 @@
+   This is part of a tech note series on identity management for the Rubin Science Platform.
+   The other primary documents are DMTN-234_, which describes the high-level design; and SQR-069_, which provides a history and analysis of the decisions underlying the design and implementation.
+   See :ref:`References <references>` for a complete list of related documents.
 
-  and
+Implementation overview
+=======================
 
-  Subsubsections
-  ^^^^^^^^^^^^^^
+The primary components of the identity management system for the Rubin Science Platform are:
 
-  To add images, add the image file (png, svg or jpeg preferred) to the
-  _static/ directory. The reST syntax for adding the image is
+#. Some external source of user authentication
+#. A repository of identity information about users (name, email, group membership, etc.)
+#. A Kubernetes service (Gafaelfawr_) which runs in each Science Platform deployment, performs user authentication, applies high-level access control rules, and provides identity information to other Science Platform services via an API
+#. A configuration of the ingress-nginx_ Kubernetes ingress controller that uses Gafaelfawr as an auth subrequest handler to enforce authentication and authorization requirements
+#. A user interface for creating and managing tokens, currently implemented as part of Gafaelfawr
 
-  .. figure:: /_static/filename.ext
-     :name: fig-label
+.. _ingress-nginx: https://kubernetes.github.io/ingress-nginx/
 
-     Caption text.
+The Science Platform in general, and specifically the last three components listed above, are deployed in a Kubernetes cluster using Phalanx_.
+The first two components are external to the Kubernetes cluster in which the Science Platform runs.
 
-   Run: ``make html`` and ``open _build/html/index.html`` to preview your work.
-   See the README at https://github.com/lsst-sqre/lsst-technote-bootstrap or
-   this repo's README for more info.
+Here is that architecture in diagram form:
 
-   Feel free to delete this instructional comment.
+.. figure:: /_static/science-platform.png
+   :name: High-level Science Platform architecture
 
-:tocdepth: 1
+   An overview of a Rubin Science Platform deployment.
+   This shows the major aspects of the Science Platform but omits considerable detail, including most supporting services, the identity management store, and some details of the Gafaelfawr architecture.
 
-.. Please do not modify tocdepth; will be fixed when a new Sphinx theme is shipped.
+As discussed in DMTN-234_, there is no single Rubin Science Platform.
+There are multiple deployments of the Science Platform at different sites with different users and different configurations.
+With respect to the identity management system, these differ primarily in the choice of the first two components.
 
-.. sectnum::
+General access deployments of the Science Platform use CILogon_ as the source of user authentication, and COmanage_ as the repository of identity information.
```

Comment:
Passive voice makes this sound inevitable, maybe "We have chosen to [] for our main general access deployment.